.github/workflows/mac.yml

```diff
@@ -0,0 +1,132 @@
+name: GraphScope CI on MacOS
+
+on:
+  # Trigger the workflow on push or pull request, but only for the main branch
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'gnn_engine/**'
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+      - 'research/**'
+      - 'scripts/**'
+      - 'tutorials/**'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'gnn_engine/**'
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+      - 'research/**'
+      - 'scripts/**'
+      - 'tutorials/**'
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    env:
+      JAVA_HOME: /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home
+      OPENSSL_ROOT_DIR: /usr/local/opt/openssl
+      OPENSSL_LIBRARIES: /usr/local/opt/openssl/lib
+      OPENSSL_SSL_LIBRARY: /usr/local/opt/openssl/lib/libssl.dylib
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Setup tmate session
+      if: false
+      uses: mxschmitt/action-tmate@v2
+
+    - name: Install Dependencies for MacOS
+      run: |
+        pushd ${GITHUB_WORKSPACE}
+        brew install llvm@9
+
+        # setup envs
+        export CC=/usr/local/opt/llvm@9/bin/clang
+        export CXX=/usr/local/opt/llvm@9/bin/clang++
+        export LDFLAGS=-L/usr/local/opt/llvm@9/lib
+        export CPPFLAGS=-I/usr/local/opt/llvm@9/include
+        export PATH=/usr/local/opt/llvm@9/bin:$PATH
+        /bin/bash ./scripts/install_dependencies.sh
+        popd
+
+    - name: Install libgrape-lite
+      run: |
+        git clone -b master --single-branch --depth=1 https://github.com/alibaba/libgrape-lite.git
+        cd libgrape-lite
+        mkdir build && cd build
+        cmake ..
+        make -j
+        sudo make install
+
+    - name: Setup tmate session
+      if: false
+      uses: mxschmitt/action-tmate@v2
+
+    - name: Install libvineyard
+      run: |
+        git clone -b main-v0.2.5 --single-branch --depth=1 https://github.com/alibaba/libvineyard.git
+        cd libvineyard
+        git submodule update --init
+        mkdir build && pushd build
+        cmake .. -DBUILD_VINEYARD_PYTHON_BINDINGS=ON -DBUILD_SHARED_LIBS=ON -DBUILD_VINEYARD_IO_OSS=ON -DBUILD_VINEYARD_TESTS=OFF
+        make -j
+        make vineyard_client_python -j
+        sudo make install
+        popd
+        python3 setup.py bdist_wheel
+        pip3 install ./dist/*.whl
+
+    - name: Build GraphScope
+      run: |
+        export PATH=/usr/local/opt/gnu-sed/libexec/gnubin:${JAVA_HOME}/bin:${PATH}:/usr/local/zookeeper/bin
+
+        make gae gie
```

Comment:
We won't enable GLE in this pull request, right?

Reply:
yes